pkg/lib/feature/verification/claim.go

```diff
@@ -12,9 +12,10 @@ const (
 )
 
 type ClaimStatus struct {
-	Name      string
-	Status    Status
-	IsEnabled bool
+	Name                       string
+	Verified                   bool
+	RequiredToVerifyOnCreation bool
+	EndUserTriggerable         bool
```

Comment:
Delete the type Status and make everything compiles again.

Reply:
I meant delete Status and its variants (StatusRequired, StatusDisabled, StatusPending, StatusVerified). Merely deleting the type does not give any benefit.

Reply:
@andychow326